src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -453,6 +454,10 @@ public async Task AddNode()
 
                     await ServerStore.AddNodeToClusterAsync(nodeUrl, nodeTag, validateNotInTopology: true, asWatcher: watcher ?? false);
 
+                    if (LoggingSource.AuditLog.IsInfoEnabled)
+                        LogAuditFor("Server", "PUT", $"Node {ServerStore.NodeTag} (url: {Server.WebUrl}) added node {nodeTag} (url: {nodeUrl}) to cluster. term: {ServerStore.Engine.CurrentTerm}.");
```

Comment:
We concatenate `action` and `target` parameters. Wouldn't it make sense if a sample log message looked as:

`ADD Node D (url: http://.....) to cluster. Term: 12323`

Reply:
Does it make sense to include `{ServerStore.NodeTag}` and `{Server.WebUrl}`?

Reply:
You right, it is unnecessary,
I'll remove it